### PR TITLE
Added better darkening to treccani.it

### PR DESCRIPTION
**Pull request by Leone25**
removed weird yellowish color from background and added better contrast

### CODE REVIEW

**Comment by Leone25** [comment]
i should have fixed it

**Comment by Leone25** [comment]
tho i don't understand the errors . ..

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -3318,6 +3318,31 @@ INVERT
 
 ================================
 
+treccani.it
+*.treccani.it
+
+CSS
+.treccani_portal .wrapper, .treccani_portal body {
+    background-color: rgb(0, 0, 0) !important;
```
[the remark this answers is not in the document]
Comment (Leone25) [reply src/config/dynamic-theme-fixes.config]: Done

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -3318,6 +3318,31 @@ INVERT
...
+    background-color: rgb(30, 30, 30) !important;
+}
+.article_template-bg-text-medium .block .block-content {
+    background-color: rgb(30, 30, 30) !important;
+}
+.module-footer .footer-colophon {
+    background-color: rgb(30, 30, 30) !important;
+}
+#nav-icon3 span {
+    background-color: rgb(255, 255, 255) !important;
```
[the remark this answers is not in the document]
Comment (Leone25) [reply src/config/dynamic-theme-fixes.config]: Done

**Comment by Gusted** [comment]
Wasn't your fault, Travis had a bit of a problem in the backend.

**Comment by Leone25** [comment]
Yea that's what I thought too, but I have no experience on this kind of bots . . . Ty anyway

**Pull request merged**